Library/Formula/conoha-iso.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class ConohaIso < Formula
+  desc "Command wrapper to use ISO of ConoHa."
+  homepage "https://github.com/hironobu-s/conoha-iso"
+  url "https://github.com/hironobu-s/conoha-iso/archive/current.tar.gz"
```

Comment:
Why does this have no version number? I'm concerned this tag is going to be changed in future.